curriculum/challenges/english/14-responsive-web-design-22/learn-css-colors-by-building-a-set-of-colored-markers/619b74fa777a2b2473c94f82.md

```diff
@@ -7,7 +7,9 @@ dashedName: step-45
 
 # --description--
 
-Update the CSS class selector `.two` so it targets the new `green` class. And update the `.three` selector so it targets the new `blue` class.
+A **class selector** is also a **CSS rule**. Don't be confused by the usage of those two terms. All CSS selectors are CSS rules too.
```

Comment:
Instead of adding this confusing explanation, couldn't we just pick one term and use it throughout the project?

Reply:
Okay